`issubclass() arg 1 must be a class` error in Pydantic

Error from CI on Python 3.8 and 3.9:

```
________ ERROR collecting boefjes/katalogus/tests/test_organisations.py ________
boefjes/katalogus/tests/test_organisations.py:6: in <module>
    from boefjes.katalogus.api import app
boefjes/katalogus/api.py:17: in <module>
    from boefjes.katalogus.storage.interfaces import StorageError
boefjes/katalogus/storage/interfaces.py:4: in <module>
    from boefjes.katalogus.models import Organisation, Repository
boefjes/katalogus/models.py:37: in <module>
    class Boefje(Plugin):
pydantic/main.py:197: in pydantic.main.ModelMetaclass.__new__
    ???
pydantic/fields.py:506: in pydantic.fields.ModelField.infer
    ???
pydantic/fields.py:436: in pydantic.fields.ModelField.__init__
    ???
pydantic/fields.py:552: in pydantic.fields.ModelField.prepare
    ???
pydantic/fields.py:668: in pydantic.fields.ModelField._type_analysis
    ???
/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/typing.py:774: in __subclasscheck__
    return issubclass(cls, self.__origin__)
E   TypeError: issubclass() arg 1 must be a class
```

--- a/boefjes/requirements-dev.txt
+++ b/boefjes/requirements-dev.txt
@@ -11,6 +11,7 @@ psycopg2==2.9.6
 pylxd==2.3.1
 pynacl==1.5.0
 sqlalchemy==1.4.48
+typing-extensions==4.5.0
 
 # Dev dependencies
 pre-commit==3.2.2

--- a/boefjes/requirements.txt
+++ b/boefjes/requirements.txt
@@ -11,6 +11,7 @@ psycopg2==2.9.6
 pylxd==2.3.1
 pynacl==1.5.0
 sqlalchemy==1.4.48
+typing-extensions==4.5.0
 python-dateutil==2.8.2
 
 # OpenTelemetry

--- a/mula/requirements.txt
+++ b/mula/requirements.txt
@@ -9,6 +9,7 @@ pydantic==1.10.7
 python-dotenv==1.0.0
 requests==2.27.1
 sqlalchemy==1.4.48
+typing-extensions==4.5.0
 uvicorn==0.17.1
 
 # OpenTelemetry

